optimum/habana/accelerate/state.py

```diff
@@ -60,10 +60,6 @@ def __init__(self, cpu: bool = False, **kwargs):
                     self.distributed_type = GaudiDistributedType.DEEPSPEED
                     import deepspeed
 
-                    if world_size > 1:
```

Comment:
These changes were added here: 
https://github.com/huggingface/optimum-habana/pull/1089/files

let's repeat your experiments in virtual env and see if indeed these are not needed. 
if the changes are not the best coded apprach, pls change them and retest.

Reply:
Ok. Thanks.